packages/trace-viewer/src/ui/networkResourceDetails.tsx

```diff
@@ -94,6 +99,10 @@ const RequestTab: React.FunctionComponent<{
         {resource.request.queryString.map(param => `${param.name}: ${param.value}`).join('\n')}
       </div>
     </> : null}
+    <div className='network-request-details-header'>Time</div>
```

Comment:
What's the point in showing these values in the details, given that they are already present in the network table where one can see other network requests and compare their start time/duration?

Reply:
While I agree with you that it's nice to see this information in the details view, I still cannot come up with a good explanation why it should be shown above request headers. Maybe push it below headers at least? We should probably put timing information into its own tab and populate it with [other](https://playwright.dev/docs/api/class-request#request-timing) details that we have, but I don't remember anyone asking for this.